front/lib/models/workspace.ts

```diff
@@ -68,6 +76,16 @@ Workspace.init(
       type: DataTypes.BOOLEAN,
       defaultValue: false,
     },
+    whiteListedProviders: {
```

Comment:
If we can reverse the approach and only set a value when admins have blacklisted some providers, it would be more efficient. Otherwise, this pattern will require running a migration every time we support new providers. While this isn’t an everyday operation, IMHO we should keep this process as straightforward as possible.

Reply:
Agreed. null is everything. Setting is restricting

Reply:
Changed to this 👍🏼